src/dotnet-serve/CommandLineOptions.cs

```diff
@@ -119,6 +119,11 @@ public virtual bool UseTls
         [Option("-z|--gzip", Description = "Enable gzip compression")]
         public bool UseGzip { get; }
 
+#if !NETCOREAPP2_1
```

Comment:
Is it supported by 2.2 then? Seems like this should check for that too

Reply:
This command line tool only compiled for 2.1 and 3.0, not 2.2, so that check isn't necessary.

Reply:
ah, of course thanks :)